DocumentedExamples/Querying_Scalar_Quantities_and_Annually_Averaged_Timeseries.ipynb

```diff
@@ -0,0 +1,2091 @@
+{
```

Comment:
<p>As commented below, I suggest just keeping the 6 standard cycles, not the extension runs. And you may as well display them in order at the same time. </p><p> </p><p>I.e. add:</p><p> </p><p><code>experiments = sorted(experiments[:6])</code></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/247/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>